Ryujinx.Headless.SDL2/Options.cs

```diff
@@ -85,6 +85,9 @@ public class Options
         [Option("fs-global-access-log-mode", Required = false, Default = 0, HelpText = "Enables FS access log output to the console.")]
         public int FsGlobalAccessLogMode { get; set; }
 
+        [Option("fps-precision", Required = false, Default = 0, HelpText = "Integer scale to adjust frame metric displayed value. Values from 0-4")]
+        public int FpsPrecision { get; set; }
```

Comment:
```suggestion
        public uint FpsPrecision { get; set; }
```

The value is between 0 and 4 so a uint should be enough